docs/3.x.x/installation-options/quick-start-detailed.md

```diff
@@ -0,0 +1,400 @@
+
+:::
+
+The next steps involve [setting roles and permissions](quick-start-detailed.html#_5-set-roles-and-permissions) for these content types. Let's do that.
+
+Click on the Strapi Logo to return to the main administration panel.
+
+![Main Admin Panel](../assets/quick-start-detailed/mainAdminPanel.jpg 'Main Admin Panel')
+
+---
+
+## 5. Set roles and permissions
+
+By default, Strapi publishes all Content Types with restricted permissions. Which means you have to explicitly give permissions to each Content Type you create. We are going to give **Public** (any web browser with the correct link) access to the **Restaurant** Content Type.
+
+Locate and click on the **Roles & Permission** menu item under **PLUGINS** on the left menu. (The **Roles & Permission** plugin can accomplish many tasks related to permissions. For now we will focus on the **Public** role.)
+
+Next, click on the **pencil** edit icon to the right of the **Public** Role.
+
+![Roles and Permissions Panel](../assets/quick-start-detailed/RolesAndPermissionsPanel.jpg 'Roles and Permissions Panel')
+
+From here, scroll down under **Permissions** and find **Restaurant**. Click the checkbox next to **find**. To the right, you will see the URL route. It should say, `/restaurants"`. Scroll back to the top, and click the **Save** button.
+
+![Check Find for Restaurant](../assets/quick-start-detailed/rolesFindAndRoute.jpg 'Check Find for Restaurant')
+
+You are returned to the **Roles and Permission** Panel.
+
+![Roles and Permissions Panel](../assets/quick-start-detailed/RolesAndPermissionsPanel.jpg 'Roles and Permissions Panel')
+
+We are now ready to [Consume the Content Type API](quick-start-detailed.html#_6-consume-the-content-type-api).
+
+Click on the Strapi Logo to return to the main administration panel.
+
+---
+
+## 6. Consume the Content Type API
+
+The Project is accessible by following the `http://localhost:1337/` link. You will see the **'Welcome'** screen.
+
+![Strapi Welcome Page](../assets/quick-start-detailed/strapiWelcomePage.jpg 'Strapi Welcome Page')
+
+What we want is the **Restaurant Content Type**. The route is `/restaurants"`. In your browser, type `http://localhost:1337/restaurants`.
+
+![Successful API call to Restaurant Content Type](../assets/quick-start-detailed/successApiCall.jpg 'Successful API call to Restaurant Content Type')
+
+::: warning NOTE
+
+If you have incorrectly or not set permissions to your content type, you will get a **"403"** permission error. See the below example.
+
+Forbidden Access Looks like this:
+
+![Forbidden Access to Restaurant Content Type](../assets/quick-start-detailed/forbiddenAccessToRestaurant.jpg 'Forbidden Access to Restaurant Content Type')
+
+:::
+
+::: tip CONGRATULATIONS
+👏 Congratulations, you have now completed the Strapi Quick Start. We invite you to [join our community](/3.x.x/getting-started/community.html). Please continue reviewing our docs and tutorials to further learn how Strapi can solve your needs.
```

Comment:
👏 Congratulations, you have now completed the tutorial showing you how to get started. We invite you to [join our community](/3.x.x/getting-started/community.html). Please continue reviewing our docs and tutorials to further learn how Strapi can solve your needs.

Reply:
Great catch! Edited.